src/witan/send/adroddiad/ncy.clj

```diff
@@ -292,7 +204,148 @@
   ;;    |   18 |                           22 |                         23 |
   ;;    |   19 |                           23 |                         24 |
   ;;    |   20 |                           24 |                         25 |
+  )
 
 
+;;; # Tests
+;;; ## Tests for ncy->age… and age…->ncy
+(rcf/tests "Age at start of reception should be 4 years, per gov.uk/schools-admissions/school-starting-age"
+           (ncy->age-at-start-of-school-year 0) := 4)
 
-  )
+(rcf/tests
```

Comment:
I think this is fine, but I didn't think (and the name leads me to believe) that you don't need each test in its own rcf/tests block.

Reply:
You don't need to put each test in it's own rcf/test block, but I found a couple of reasons why it was better:
1. With multiple tests when one fails the description strings for all tests up to and including the failing test are written out to the repl, making it hard to see exactly what test has failed. (I suspect this is not the intended functionality?)
1. It makes it easier to run one test at a time during development.
2. The (rcf/tests) call returns the value of the last expression, which again helps during development. (Though only have to move the cursor one paren in to get that so a minor point!)

If we can fix #1 then #2 and #3 not sufficient to justify in themselves, but for now I think better to have separate to address #1.